README.md

```
@@ -1,177 +1,441 @@
+### 4.1 사전 요구사항
+
+- Python 3.11+
+- Node.js 20+
+- AWS 계정 + IAM (S3, DynamoDB, OpenSearch, RDS 등)
+- OpenAI API 키
+- PostgreSQL 인스턴스 (RDS 또는 로컬)
+
+### 4.2 레포 클론
+
+bash
+git clone https://github.com/ORG/REPO.git
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)